queries/lua/injections.scm

```diff
@@ -135,6 +135,12 @@
   (#set! injection.language "luadoc")
   (#offset! @injection.content 0 1 0 0))
 
+(comment
+  content: (_) @injection.content
+  (#lua-match? @injection.content "^[-][%s]*[|]")
```

Comment:
I mean, is there a reason for this `[|]` to not be put with the query pattern right above it? :thinking:

Reply:
lua match doesn't have an OR operator?

Reply:
I don't see why this pattern won't work
```
(comment
  content: (_) @injection.content
  (#lua-match? @injection.content "^[-][%s]*[|@]")
  (#set! injection.language "luadoc")
  (#offset! @injection.content 0 1 0 0))
```

Reply:
right...